lisp/Makefile

```diff
@@ -39,6 +39,7 @@ dist: all
 
 install: dist
 	-$(INSTALLDIR) $(LISPDIR)
+	$(INSTALL) $(EL) $(LISPDIR)
```

Comment:
$(ELS), perhaps, or am I missing something.

Reply:
That autoloads filter in ELS is not triggered on the clean repo. When ess-autoloads.el is in the repo the elc is not generated. I think `batch-byte-compile` just skips it or something.